docs/content/foundations/typography/text-styles.mdx

```diff
@@ -4,145 +4,285 @@ page_title: Text Styles
-| A line of text and <span className="display-inline-block width-card padding-1 border-1px"><span className="text-wrap">.text-wrap formatted line with spaces and stuff</span></span>         |
-| A line of text and <span className="display-inline-block width-card padding-1 border-1px"><span className="text-no-wrap">.text-no-wrap formatted line with spaces and stuff</span></span>   |
-
-## Font Weight Tokens
-
-Font weights control the heaviness or lightness of how a font displays. The NCIDS supports 3 different weights.
-
-<FontWeightTable>
-	<FontWeightRow
-		token="light"
-		weight="300"
-		variable="theme-font-weight-light"
-	/>
-	<FontWeightRow
-		token="normal"
-		weight="400"
-		variable="theme-font-weight-normal"
-	/>
-	<FontWeightRow token="bold" weight="600" variable="theme-font-weight-bold" />
-</FontWeightTable>
-
-### Font Weight Utilities
-
-To set the weight of the font.
-
-Utilities may be used in component Sass with utility mixins.
-
-Highlighted `tokens` are the design tokens that serve as the foundation for all design system styles.
-
-| Utility        | Mixin            | Example                     |
-| :------------- | :--------------- | :-------------------------- |
-| .text-`weight` | u-text(`weight`) | `@include u-text('normal')` |
+utility_modules:
+  ## The human readable name of the module.
+  - name: Font style
+    utility_module_name: 'font-style'
+    utility_class_display_component: 'TextDemonstration'
+    utility_class_display_params:
+      display_text: short
+      classes:
+        - class_name: 'text-italic'
+          calculated_value: 'italic'
+        - class_name: 'text-no-italic'
+          calculated_value: 'normal'
+  - name: Font weight
+    utility_module_name: 'font-weight'
+    utility_class_display_component: 'TextDemonstration'
+    utility_class_display_params:
+      display_text: short
+      classes:
+        - class_name: 'text-light'
+          calculated_value: '300'
+        - class_name: 'text-normal'
+          calculated_value: '400'
+        - class_name: 'text-bold'
+          calculated_value: '600 changed from the default of 700'
+  - name: Letterspacing
+    utility_module_name: 'letter-spacing'
+    utility_class_display_component: 'TextDemonstration'
+    utility_class_display_params:
+      display_text: short
+      classes:
+        - class_name: 'text-ls-neg-3'
+          calculated_value: '-0.03em'
+        - class_name: 'text-ls-neg-2'
+          calculated_value: '-0.02em'
+        - class_name: 'text-ls-neg-1'
+          calculated_value: '-0.01em'
+        - class_name: 'text-ls-auto'
+          calculated_value: 'initial'
+        - class_name: 'text-ls-1'
+          calculated_value: '-0.025em'
+        - class_name: 'text-ls-2'
+          calculated_value: '-0.1em'
+        - class_name: 'text-ls-3'
+          calculated_value: '-0.15em'
```

Comment:
@karlikpj These calculated values are incorrect. Please be more careful - not reviewing work after copy/pasting slows down the reviews and acceptance of these PRs.